integration/actions_test.go

```diff
@@ -134,37 +135,65 @@ func TestBuildActionPipeline(t *testing.T) {
 		t.Fatalf("Create namespace action failed: %s", err.Error())
 	}
 
-	dir, err := os.MkdirTemp("", "")
+	cwd, err := os.Getwd()
 	if err != nil {
 		t.Fatal(err)
 	}
-	log.Printf("created tempdir: %s", dir)
-	dockerfilePath := filepath.Join(dir, "Dockerfile")
+	tempDir, err := os.MkdirTemp("", "")
+	if err != nil {
+		t.Fatal(err)
+	}
+	log.Printf("created tempdir: %s", tempDir)
+	dockerfilePath := filepath.Join(tempDir, "Dockerfile")
 	dockerfileContent := []byte("FROM alpine")
 	if err := os.WriteFile(dockerfilePath, dockerfileContent, 0644); err != nil {
 		t.Fatal(err)
 	}
+	manifestPath := filepath.Join(tempDir, "okteto.yml")
+	manifestContent := []byte(`
+build:
+  alpine:
+    context: .
+`)
+	if err := os.WriteFile(manifestPath, manifestContent, 0644); err != nil {
+		t.Fatal(err)
+	}
 
 	actionRepo := fmt.Sprintf("%s%s.git", githubHttpsUrl, buildPath)
-	actionFolder := strings.Split(buildPath, "/")[1]
+	actionFolder := path.Join(cwd, strings.Split(buildPath, "/")[1])
 	log.Printf("cloning build action repository: %s", actionRepo)
 	if err := cloneGitRepo(ctx, actionRepo); err != nil {
 		t.Fatal(err)
 	}
 	log.Printf("cloned repo %s \n", actionRepo)
 	defer deleteGitRepo(ctx, actionFolder)
 
-	log.Printf("building image")
 	command := fmt.Sprintf("%s/entrypoint.sh", actionFolder)
 
-	args := []string{fmt.Sprintf("okteto.dev/%s:latest", namespace), dockerfilePath, dir}
-
-	cmd := exec.Command(command, args...)
-	cmd.Env = os.Environ()
-	o, err := cmd.CombinedOutput()
-	if err != nil {
-		t.Fatalf("%s %s: %s", command, strings.Join(args, " "), string(o))
-	}
+	t.Run("okteto build with dockerfile", func(t *testing.T) {
+		// test v1 build with dockerfile
+		log.Printf("building image")
+		args := []string{fmt.Sprintf("okteto.dev/%s:latest", namespace), dockerfilePath, tempDir}
+		cmd := exec.Command(command, args...)
+		cmd.Env = os.Environ()
+		cmd.Dir = tempDir
+		o, err := cmd.CombinedOutput()
+		if err != nil {
+			t.Fatalf("%s %s: %s", command, strings.Join(args, " "), string(o))
+		}
+	})
+	t.Run("okteto build with manifest", func(t *testing.T) {
```

Comment:
@ifbyol the `cli-integration` is running a fix version? I thought it was running master to detect if we are breaking the current version of the CLI.

Reply:
@pchico83  `cli-integration` is running the version of the CLI we were installing in the circleci containers that was the one from https://get.okteto.com, and now we use the CLI from the `golang-ci` image. The purpose of that job was to be sure we don't break the latest CLI released with the changes released on `app` master.

Just to clarify, I'm talking about this job https://github.com/okteto/app/blob/master/.circleci/config.yml#L181

Reply:
@ifbyol makes sense. In that case, that job should clone the labels corresponding to the CLI under test, no?

Reply:
@pchico83  yep, but the actions are being cloned in the tests, so we would have to modify these tests.

Apart from that, we have to bear that in mind for the kind of tests v1 vs v2

Reply:
Ah, I see your point. Yes, it's problematic :-)

Reply:
@ifbyol I would go with testing `master` on `cli-integration`. We have other processes to test previous versions

Reply:
@pchico83 but we wouldn't have nothing to verify that what we are releasing to Cloud is not breaking anything with the current CLI. We included those tests because we broke it a couple of times.

If we want to do that, we have to revert then the changes we did in https://github.com/okteto/app/pull/3540/files and install the CLI from master in every circlci job. Not a problem, only mentioning what we would have to do cc @AgustinRamiroDiaz

Reply:
i've added a skip condition for the manifest test. only cli versions 2 or higher will perform the test using the manifest, is this correct?

Reply:
yes, that would be correct @teresaromero